docs/recipes/typescript.md

```diff
@@ -51,7 +51,7 @@ import test from 'ava';
 
 const fn = async () => Promise.resolve('foo');
 
-test(async t => {
+test('receive foo asynchronously', async t => {
```

Comment:
Perhaps change this to `'fn() returns foo'`, and then make `fn` and the test synchronous. I'm not sure why this `fn` was ever asynchronous but `async () => Promise.resolve()` is rather odd.

Reply:
Okay, is now

```ts
import test from 'ava';

const fn = () => Promise.resolve('foo');

test('fn() returns foo', t => {
	t.is(fn(), 'foo');
});
```

Reply:
> I think the example may be better if fn() is synchronous, what do you think?

I actually think maybe it should have both.  `async` is still pretty hard for a lot of devs, and it might be nice for novices looking for help to get it here.  Novices have a hard time locating things, and in my opinion benefit from extra examples peppered everywhere

But also I don't care very much, I was just fixing a bug 😀

Reply:
I'll do the flow recipe in a different PR

Reply:
specifically [here](https://github.com/avajs/ava/compare/master...StoneCypher:patch-2)

Reply:
um apparently i forgot to make the actual pr

[fixed](https://github.com/avajs/ava/pull/2104)